src/server/scripts/Northrend/Ulduar/Ulduar/boss_assembly_of_iron.cpp

```diff
@@ -89,9 +89,8 @@ enum eEnums
     EVENT_LIGHTNING_TENDRILS    = 24,
     EVENT_LIGHTNING_LAND        = 25,
     EVENT_LAND_LAND             = 26,
-    EVENT_IMMUNE                = 27,
 
-    EVENT_ENRAGE                = 30,
+    EVENT_ENRAGE                = 30
```

Comment:
where are these value taken from?

Reply:
@FrancescoBorzi What values exactly? 🤔  on that part which you've marked, I just deleted the **,**  because that event is the last one so is not needed to have a **,** at the end of it. If you are taking about another thing, let me know exactly 😸